src/libs/actions/Task.js

```diff
@@ -184,113 +184,133 @@ function createTaskAndNavigate(currentUserEmail, currentUserAccountID, parentRep
     Navigation.dismissModal(optimisticTaskReport.reportID);
 }
 
-function completeTask(taskReportID, taskTitle) {
-    const message = `completed task: ${taskTitle}`;
-    const completedTaskReportAction = ReportUtils.buildOptimisticTaskReportAction(taskReportID, CONST.REPORT.ACTIONS.TYPE.TASKCOMPLETED, message);
+/**
+ * Completes an open task
+ * @param {Object} report
+ */
+function completeTask(report) {
+    const message = `completed task: ${ReportUtils.getReportName(report)}`;
```

Comment:
this looks odd to me, i don't think we should save "completed task:" to the message. It's not translated. Instead, i think the component should use another key to know if the task is completed or not and then translate and show the appropriate text. Thoughts?

Reply:
Makes total sense, I had a [similar discussion](https://expensify.slack.com/archives/C04QEB4MJEQ/p1687385149610769) about this on the API side but didn't think of changing these lines 🤦‍♂️

I think we can use [this](https://github.com/Expensify/App/blob/c122266f96bab047cda55f0cebff8b3a61e97700/src/languages/en.js#L1292-1294) to localize that message so I'll try doing that